_data/latest_version.yml

```diff
@@ -6,8 +6,19 @@ version: 2022.12.26
 # the version
 git_tag: dosbox-x-v2022.12.26
 
-win32_setup_file: dosbox-x-win32-2022.12.26-setup.exe
-win64_setup_file: dosbox-x-win64-2022.12.26-setup.exe
+#Original
+#win32_setup_file: dosbox-x-win32-2022.12.26-setup.exe
+#win64_setup_file: dosbox-x-win64-2022.12.26-setup.exe
+#winXP_setup_file: dosbox-x-winXP-2022.12.26-setup.exe
+
+# Change link to portable files to at least avoid 404 error,
+# since 2022.12.26 installers are not available.
+# Installers should be available in the next release, and the linked to them.
```

Comment:
```suggestion
# The actual installers should be linked in the next release,
# where they are available.
```
I think we probably should get the installers for 2022.12.26 built and uploaded to the release, as it's still possible